errors.go

```diff
@@ -0,0 +1,18 @@
+package e5e
+
+import "fmt"
+
+// InvalidEntrypointError is returned if the given entrypoint did not get registered on runtime.
+type InvalidEntrypointError struct{ Entrypoint string }
+
+func (e InvalidEntrypointError) Error() string {
+	return fmt.Sprintf("entrypoint %q does not exist", e.Entrypoint)
+}
+
+// EntrypointRuntimeError is a generic error for everything that happens during runtime.
+type EntrypointRuntimeError struct{ inner error }
```

Comment:
This error gets used for errors returned from the entrypoint and our runtime. This should be clearly distinguishable. 
Do the separate error types really provide an advantage for us? The old error fmt chaining was used to have a kind of "stacktrace" to make debugging errors in e5e executions somehow tracable as at the end only the final panic call is available.